chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2631,7 +2563,7 @@ public void setUseCustomWaterColor(boolean value) {
       ((Configurable) postProcessingFilter).storeConfiguration(postprocessJson);
       json.add("postprocessSettings", postprocessJson);
     }
-    json.add("outputMode", outputMode.getName());
+    json.add("outputMode", pictureExportFormat.getName());
```

Comment:
Make sure to keep the old names to not make this a breaking change for existing scene files.